Python/Cryptocurrency_Converter/crypto_converter.py

```diff
@@ -1,110 +1,95 @@
 """we will be using coinbase API as it is public
-More info below
 https://developers.coinbase.com/api/v2?python#get-exchange-rates"""
 
 import requests
+import sys
 
 
-def exchange_rates(cryptocurrency):
-    """
-    This function gets exhange rates.
-    The coinbase API returns the JSON for an input cryptocurrency with
-    data or exhangerate of one coin to all known "normal" fiat currencies
-    like BTC(input) to INR,USD,etc.
-    """
-
-    # capitalising the cryptocurrency values as the API accepts and returns
-    cryptocurrency = cryptocurrency.upper()
-
-    # These are the cryptocurrency available currently by the API.
-    # "BTC" is for Bitcoin,"ETH" is for Ethereum and so on
-    available_cryptocurrencies = [
-        'BTC', 'ETH', 'ETC', 'BCH', 'LTC', 'ZEC', 'ZRX']
-
-    if cryptocurrency not in available_cryptocurrencies:
-        return None
+def exchange_rates(typee):
+    """This function gets exhange rates. The coinbase API returns the JSON for an input cryptocurrency with data or exhangerate of one coin to all known "normal" fiat currencies like BTC(input) to INR,USD,etc."""
+    typee = typee.upper()
+    response_obj = requests.get('https://api.coinbase.com/v2/exchange-rates?currency='+typee)
 
-    # sending a request to the api and storing the response object in r variable
-    response_obj = requests.get(
-        'https://api.coinbase.com/v2/exchange-rates?currency='+cryptocurrency)
-
-    try:
-        exchange_dict = response_obj.json()["data"]["rates"]
-    except KeyError:
-        return None
+    try: exchange_dict = response_obj.json()["data"]["rates"]
+    except: return None
     return exchange_dict
 
-# This function converts currency amount to cryptocurrency amount
-
-
-def convert_currency_to_crypto(amount, currency, cryptocurrency):
-    """This function converts currency amount to cryptocurrency amount"""
-
-    # checking that the input is correct
-    try:
-        amount = float(amount)
-    except ValueError:
-        return "wrong input!"
-
+def convert_fiat_to_crypto(fvalue, ftype, ctype):
+    """This function converts fiat money to cryptocurrency amount."""
```

Comment:
Brief information about the paramters `ftype`, `fvalue` and `ctype` can be included in the converter function itself.